toolkits/springer/packages/springer-author-list/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@springernature/springer-author-list",
-  "version": "0.1.0",
+  "version": "1.0.0",
```

Comment:
Is this a breaking change for author-list?

Reply:
Going to bump it to 0.2.0 instead and for the others to minor rather than major version as there were no breaking changes 😬